lib/glific.ex

```diff
@@ -60,6 +60,26 @@ defmodule Glific do
         )
   end
 
+  @doc """
+  converts list to indexed map
+  """
+  @spec to_indexed_map(list(), integer()) :: map()
+  def to_indexed_map(list, offset \\ 1)
+      when is_list(list) and
+             is_integer(offset) do
+    Enum.reject(list, fn list_item -> list_item in ["", nil] end)
+    |> do_to_indexed_map(offset)
+  end
```

Comment:
Let's remove this function all togenther. I don't see any other use case to convert add indexes in the map like this. If we have more than one use case then we can add it.